ChopChop/Storage/Models/Recipes/RecipeStepRecord.swift

```diff
@@ -2,19 +2,27 @@ import GRDB
 
 struct RecipeStepRecord {
     var id: Int64?
-    var recipeId: Int64?
-    var index: Int
+    var graphId: Int64?
     var content: String
 }
 
 extension RecipeStepRecord: Codable, FetchableRecord, MutablePersistableRecord {
     enum Columns {
         static let id = Column(CodingKeys.id)
-        static let recipeId = Column(CodingKeys.recipeId)
-        static let index = Column(CodingKeys.index)
+        static let graphId = Column(CodingKeys.graphId)
         static let content = Column(CodingKeys.content)
     }
 
+    static let outgoingEdges = hasMany(RecipeStepEdgeRecord.self)
+    var outgoingEdges: QueryInterfaceRequest<RecipeStepEdgeRecord> {
+        request(for: RecipeStepRecord.outgoingEdges)
+    }
+
+    static let incomingEdges = hasMany(RecipeStepEdgeRecord.self)
+    var incomingEdges: QueryInterfaceRequest<RecipeStepEdgeRecord> {
+        request(for: RecipeStepRecord.incomingEdges)
+    }
```

Comment:
What is this for?

Reply:
I wasn't sure where to express these two associations. Would it be better to have something (inside `RecipeStepEdgeRecord.swift` like
```
static let source = hasOne(RecipeStepRecord.self)
var source: QueryInterfaceRequest<RecipeStepRecord> {
    request(for: RecipeStepEdgeRecord.source)
}
```

Actually is there even a need to express this association, if we're not using it?